x-pack/plugins/index_management/server/lib/get_managed_templates.ts

```diff
@@ -5,16 +5,20 @@
  * 2.0.
  */
 
+import { IScopedClusterClient } from 'kibana/server';
+
 // Cloud has its own system for managing templates and we want to make
 // this clear in the UI when a template is used in a Cloud deployment.
 export const getCloudManagedTemplatePrefix = async (
-  callAsCurrentUser: any
+  client: IScopedClusterClient
 ): Promise<string | undefined> => {
   try {
-    const { persistent, transient, defaults } = await callAsCurrentUser('cluster.getSettings', {
-      filterPath: '*.*managed_index_templates',
-      flatSettings: true,
-      includeDefaults: true,
+    const {
+      body: { persistent, transient, defaults },
+    } = await client.asCurrentUser.cluster.getSettings({
```

Comment:
I didn't test this -- could someone look into setting these settings manually and verifying there are no errors?

Reply:
Implemented via https://github.com/elastic/kibana/pull/43901.

Looks like it is working as expected for both composable and legacy index templates.

<img width="1365" alt="Screen Shot 2021-07-26 at 12 10 03 PM" src="https://user-images.githubusercontent.com/5226211/127022472-ccfa4020-55bc-4420-8c03-14da219edba4.png">
<img width="1363" alt="Screen Shot 2021-07-26 at 12 10 09 PM" src="https://user-images.githubusercontent.com/5226211/127022474-66c0a679-7e46-49c5-a881-dbea45565862.png">